### ISSUE TITLE
Change the e-mail address associated with an account

### ISSUE THREAD

**Issue by schoen** [certbot/certbot#1215]
#1214 was originally discovered as a result of a user wanting to change the e-mail address associated with an account and not knowing how to do that. I don't think there's currently a documented way to do this, but there should be. This probably also requires changes on the boulder side and maybe an additional API...?

**Comment by sushain97** [comment]
+1

**Comment by rolandshoemaker** [comment]
The API to do this already exists on the boulder side, POSTing a registration object with a changed contacts field to /acme/reg should update the registration.

**Comment by mattmccutchen** [comment]
In case anyone comes here wanting to change their email address manually, I was able to do this by abusing the official client (tested on the test server).  The procedure:

1. Complete the instructions in [Running a local copy of the client](https://letsencrypt.readthedocs.org/en/latest/contributing.html#running-a-local-copy-of-the-client).
2. Temporarily grant the current user permissions on `/etc/letsencrypt` and `/var/log/letsencrypt`.
3. In `letsencrypt/cli.py`, in `obtain_cert`, just after `le_client = _init_le_client(...)`, insert `import pdb; pdb.set_trace()`.
4. Run: `letsencrypt certonly --standalone`
5. When you hit the breakpoint, run: `le_client.acme.update_registration(le_client.account.regr, client.messages.Registration.from_data(email='NEW_EMAIL_ADDRESS'))`.  Ignore the `UnexpectedUpdate` error.
6. Run `le_client.acme.query_registration(le_client.account.regr)` and make sure it shows the new email address.
7. Press Ctrl-D.  The debugger will exit without completing the original command.
8. Change the permissions back.

With a little more effort, this could probably be turned into a first-draft Python script, but I've spent enough time on it for now.
